src/pages/PetPage/PetForm.jsx

```
@@ -0,0 +1,345 @@
+  const [weightBack, setWeightBack] = useState(
+    basicData.weight ? basicData.weight.split('.')[1] : '',
+  );
+  const [previewUrl, setPreviewUrl] = useState(''); // 미리보기 URL 상태
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (!checkDateForm()) return;
+    const today = new Date();
+    const inputDate = new Date(birthDate);
+    const isValidDate =
+      /^\d{4}-\d{2}-\d{2}$/.test(birthDate) && !isNaN(inputDate);
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)